### DISCUSSION TITLE
Is it for beginners?

### DISCUSSION THREAD

**Discussion by aeb-dev** [nxrighthere/UnrealCLR#10] [General]
Hello,

First of all your project made me very excited, thank you for your hard work.

I have joined to the gaming club recently with Unity because I know c#. How easy it is for a newbie to start learning Unreal with UnrealCLR who is experienced in C#?

Also how complete are the api and docs?

**Comment by nxrighthere** [comment]
Assuming that we are comparing it with Unity, the learning curve is about the same, but you have more freedom here in terms of toolset and tight connection with the .NET ecosystem. 

In terms of API completeness, please, see the [roadmap](https://github.com/users/nxrighthere/projects/5?fullscreen=true) and [API reference](https://github.com/nxrighthere/UnrealCLR/blob/master/API/UnrealEngine-Framework.md). Generally speaking, the framework can be used for making games in the current state, but since the plugin is in the preview stage, things might change significantly.

**Comment by aeb-dev** [comment]
Thank you for the detailed answer. I will stop by with .NET 5.
